frontend/src/features/properties/components/forms/ParcelDetailForm.test.tsx

```diff
@@ -265,7 +265,8 @@ describe('ParcelDetailForm', () => {
       await fillInput(container, 'isSensitive', true, 'radio');
       // await fillInput(container, 'financials.1.date', exampleData.evaluations[1].date);
       // TODO: add a function capable of filling this type of field
-      await fillInput(container, 'financials.0.netbook.value', exampleData.fiscals[1].value);
+      // Net Book no longer required
+      // await fillInput(container, 'financials.0.netbook.value', exampleData.fiscals[1].value);
```

Comment:
I think at this point we shouldn't just comment it out.

Reply:
Ah ok, I will update it!